src/services/exports/exports.repository.ts

```diff
@@ -928,41 +929,41 @@ const generateCsvHistoriqueDossiersConvention = async (
 ) => {
   try {
     const fileHeaders = [
-      'Id de la structure',
-      'Nom de la structure',
-      'Type de la structure',
-      'Région',
-      'Département',
-      'Statut',
-      'Nombre de postes attribués',
-      'Date de la demande',
+      'ID',
+      'Siret plateforme',
+      'Date COSELEC',
+      'Nombre de postes avant COSELEC',
+      'Nombre de postes après COSELEC',
+      'Variation',
       'Type de la demande',
-      'Nombre de CNFS souhaités',
-      'Nombre de contrat validés',
-      'Nombre de postes renouvelés',
-      'Date de fin du premier contrat',
-      'État de la demande',
+      'N° DS',
+      'Code département',
+      'Département',
+      'Région',
+      'Type de conventionnement',
     ];
 
     res.write(
       [
         fileHeaders.join(csvCellSeparator),
         ...structures.map((structure) =>
           [
-            structure?.idPG,
-            structure?.nom,
-            structure?.type === 'PRIVATE' ? 'privée' : 'publique',
-            structure?.codeRegion,
-            structure?.codeDepartement,
-            structure?.statutStructure,
-            structure?.nbPostesAttribuees ?? 'Non renseigné',
-            formatDate(structure?.dateSorted),
-            structure?.statut,
-            structure?.nbPostesSouhaites ?? '',
-            structure?.nbContratsValides ?? 'Non renseigné',
-            structure?.nbContratsRenouveles ?? 'Non renseigné',
-            formatDate(structure?.dateFinPremierContrat),
-            structure?.statutDemande,
+            structure.idPG,
+            structure.siret,
+            structure.dateDeValidation instanceof Date
+              ? formatDate(structure.dateDeValidation)
+              : 'Non renseignée',
```

Comment:
pas besoin de ça la fonction formatDate le fait tout seul

Reply:
j'ai rajouté instanceof Date dans le fonction car j'ai un message d'erreur TypeError: date.getTime is not a function